packages/core/helper-plugin/src/features/GuidedTour.tsx

```diff
@@ -56,12 +64,20 @@ const GuidedTourContext = React.createContext<GuidedTourContextValue>({
       create: false,
       success: false,
     },
-    transferTokens: {
```

Comment:
same here, you can't remove it in v4.

Reply:
I think it wasn't used at all no?
Same question, what should I do then?

Reply:
You need to keep it for now, but if you want to remove it in `v5` add a note for `deprecated` 👍🏼